dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/DefaultEnrollmentService.java

```diff
@@ -102,6 +102,7 @@ public Enrollment getEnrollment(
     }
 
     Enrollment result = new Enrollment();
+    result.setId(enrollment.getId());
```

Comment:
Why do we need to set these?

Reply:
Because the store is working with Hibernate objects and if we do not set the id then it will think that the entities are new ones and it will fail.
Not a fan of this solution but I really do not want to change the repository layer now